services/src/main/java/org/exoplatform/addons/gamification/rest/RealizationsRest.java

```diff
@@ -89,6 +91,10 @@ public Response getAllRealizations(@ApiParam(value = "result fromDate", required
       return Response.status(Response.Status.BAD_REQUEST).entity("Dates must not be blank").build();
     }
     RealizationsFilter filter = new RealizationsFilter();
+    Identity identity = ConversationState.getCurrent().getIdentity();
+    boolean isAdministrator = identity != null ? identity.isMemberOf("/platform/administrators") : false;
+    filter.setUserId(getCurrentUser());
+    filter.setAdministrator(isAdministrator);
```

Comment:
Shouldn't add the parameter of `administrator` check here, you should make this check in service layer and delete it from Filter DTO.